di-ipv-credential-issuer-stub/src/main/resources/templates/authorize.mustache

```diff
@@ -368,6 +380,26 @@
             $("#jsonPayload").val(JSON.stringify(payload, undefined, 4))
         });
     });
+
+    var evidenceDropdownData = {{{cri_stub_evidence_payloads}}}
```

Comment:
this isn't end-user facing code, but if it was, I'd be moaning about a needless (AFAICS) dependency on JavaScript to build the contents of a dropdown ;-)